samples/System.IO.Pipelines.Samples/System.IO.Pipelines.Samples.csproj

```diff
@@ -21,5 +21,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Server.Abstractions" Version="1.1.0" />
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.0" />
+    <PackageReference Include="System.Net.NameResolution" Version="4.4.0-beta-24913-02" />
```

Comment:
It is for Dns [here](https://github.com/dotnet/corefxlab/blob/00f62cb733560d6667e897d34c54df5cae205b1c/samples/System.IO.Pipelines.Samples/HttpClient/LibuvHttpClientHandler.cs#L277).

This resolves the build error: The name 'Dns' does not exist in the current context.

4.3.0 should work: https://www.nuget.org/packages/System.Net.NameResolution/